.github/workflows/certora-prover.yml

```
@@ -0,0 +1,47 @@
+name: Certora Prover Workflow
+
+on:
+  push:
+    branches:
+      - certora
+      - master-rules
+  pull_request:
+    branches:
+      - master
+      - master-report
+      - certora-report
```

A minor thing, since they'll probably rewrite this part, I would change this to just:
```suggestion
  pull_request:
    branches:
      - master
```